src/UserService.Business/Commands/Education/CreateEducationCommand.cs

```diff
@@ -1,67 +1,135 @@
+    private readonly IEducationRepository _educationRepository;
+    private readonly ICreateEducationRequestValidator _validator;
+    private readonly IRequestClient<ICreateImagesRequest> _createImagesRequest;
+    private readonly ILogger<CreateEducationCommand> _logger;
+
+    public CreateEducationCommand(
+    IAccessValidator accessValidator,
+    IHttpContextAccessor httpContextAccessor,
+    IDbUserEducationMapper mapper,
+    IUserRepository userRepository,
+    IEducationRepository educationRepository,
+    ICreateEducationRequestValidator validator,
+    IRequestClient<ICreateImagesRequest> createImagesRequest,
+    ILogger<CreateEducationCommand> logger)
     {
-        private readonly IAccessValidator _accessValidator;
-        private readonly IHttpContextAccessor _httpContextAccessor;
-        private readonly IDbUserEducationMapper _mapper;
-        private readonly IUserRepository _userRepository;
-        private readonly IEducationRepository _educationRepository;
-        private readonly ICreateEducationRequestValidator _validator;
-
-        public CreateEducationCommand(
-            IAccessValidator accessValidator,
-            IHttpContextAccessor httpContextAccessor,
-            IDbUserEducationMapper mapper,
-            IUserRepository userRepository,
-            IEducationRepository educationRepository,
-            ICreateEducationRequestValidator validator)
-        {
-            _accessValidator = accessValidator;
-            _httpContextAccessor = httpContextAccessor;
-            _mapper = mapper;
-            _userRepository = userRepository;
-            _educationRepository = educationRepository;
-            _validator = validator;
-        }
+      _accessValidator = accessValidator;
+      _httpContextAccessor = httpContextAccessor;
+      _mapper = mapper;
+      _userRepository = userRepository;
+      _educationRepository = educationRepository;
+      _validator = validator;
+      _createImagesRequest = createImagesRequest;
+      _logger = logger;
+    }
+
+    public async Task<OperationResultResponse<Guid>> Execute(CreateEducationRequest request)
+    {
+      Guid senderId = _httpContextAccessor.HttpContext.GetUserId();
+
+      OperationResultResponse<Guid> response = new();
+
+      if (!_accessValidator.HasRights(Rights.AddEditRemoveUsers) && senderId != request.UserId)
+      {
+        _httpContextAccessor.HttpContext.Response.StatusCode = (int)HttpStatusCode.Forbidden;
+
+        response.Errors.Add($"User {senderId} doesn`t have enough rights.");
+        response.Status = OperationResultStatusType.Failed;
+      }
+
+      _validator.ValidateAndThrowCustom(request);
+
+      List<AddImageRequest> requestImages = request.Images;
+      List<Guid> createdImagesIDs = requestImages is not null && requestImages.Any() ? 
+        await CreateImages(request, response.Errors) : new();
+      DbUserEducation dbEducation = _mapper.Map(request, createdImagesIDs);
```

Comment:
you don't need this variable

Reply:
i need 
1)_educationRepository.Add(dbEducation);
2)response.Body = dbEducation.Id;